src/index.keyboardNav.test.js

```diff
@@ -162,7 +162,7 @@ test('should set current focus as selected on tab out for simpleSelect', t => {
   t.deepEqual(wrapper.state().tags[0].label, 'ccc 1')
 })
 
-test('should scroll on keyboard navigation', t => {
+test.skip('should scroll on keyboard navigation', t => {
```

Comment:
Do you plan to fix these tests? I think these tests are still valid. In addition, we can probably add a test to ensure that old janky behavior doesn't happen.

Reply:
Fixed

Reply:
Great! Overall the changes look good. I'll try it locally and get it release soon (expect end of the week-ish).

Reply:
Sounds good, Thanks!